inference-engine/src/transformations/src/transformations/low_precision/clamp.cpp

```diff
@@ -0,0 +1,97 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/low_precision/clamp.hpp"
+#include <algorithm>
+#include <memory>
+#include <ngraph/ngraph.hpp>
+#include "transformations/low_precision/network_helper.hpp"
+
+namespace ngraph {
+namespace pass {
+namespace low_precision {
+
+ClampTransformation::ClampTransformation(const Params& params) : LayerTransformation(params) {}
+
+void ClampTransformation::registerMatcherIn(GraphRewrite& pass, TransformationContext& context) const {
+    addPattern(pass,
+               context,
+               make_op_pattern<opset1::Clamp>({ make_op_label<opset1::Multiply>() }));
```

Comment:
And once again a lot of custom stuff is used for pattern matching. Can it be unified with other transformations?